Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md

```diff
@@ -0,0 +1,61 @@
+Use the AutoFocus Feeds integration to fetch indicators from AutoFocus.
+## Configure AutoFocus Feed on Demisto
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for AutoFocus Feed.
+3. Click **Add instance** to create and configure a new integration instance.
+
```

Comment:
Where are the services here?

Reply:
will add an explanation in the table below - with the param